woocommerce/woocommerce
# Apply lint fixes to payment method integration documentation

### Changes proposed in this Pull Request:

This PR updates lint issues in the `plugins/woocommerce-blocks/docs/third-party-developers/extensibility/checkout-payment-methods/payment-method-integration.md` file.

I ignored the lint errors relating to inline HTML in the table as there is no way to display a list in a markdown table without using it.

The other issues relating to duplicate headings were addressed.

This will help #48219 merge as lint will not fail there after this is merged.



### How to test the changes in this Pull Request:

Double check the tables with lists inside display OK and  click each link in the table of contents and ensure it links to the correct section.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
